Std/Lean/Position.lean

```diff
@@ -4,8 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 -/
 import Lean.Syntax
-import Lean.Data.Lsp.Utf16
 import Lean.Meta.Tactic.TryThis
+import Lean.Data.Lsp.Utf16
```

Comment:
Is this import reordering significant?

Reply:
I think that's an accidental diff.

Reply:
Turns out the import was no longer needed.

Reply:
@joehendrix There's a bunch of other code additions that are just small changes to imports in a file. They might need a quick check to see if they are also no longer needed.